charts/logging-operator/values.yaml

```diff
@@ -226,28 +226,78 @@ logging:
   #           type: json
   #         default_topic: topic
 
-  # -- EventTailer config
-  eventTailer: {}
-    # name: sample
-    # pvc:
-    #   accessModes:
-    #     - ReadWriteOnce
-    #   volumeMode: Filesystem
-  #   storage: 1Gi
-  #   storageClassName: standard
-
-  # -- HostTailer config
-  hostTailer: {}
-    # name: sample
-    # fileTailers:
+  # EventTailer config
+  eventTailer:
+    enabled: false
+    name: event-tailer
+    image:
+      # -- repository of eventTailer image
+      repository:
+      # -- tag of eventTailer image
+      tag:
+      # -- pullPolicy of eventTailer image
+      pullPolicy:
+      # -- imagePullSecrets of eventTailer image
+      imagePullSecrets:
```

Comment:
I think `imagePullSecrets` type should be `list` not `string`.

It applies to other places as well.

Reply:
you mean the default should be that way?
```suggestion
      imagePullSecrets: []
```

Reply:
Yes, currently the generated README tells it's a string, because the default value is not explicitly defined.